src/components/App.tsx

```diff
@@ -8,19 +8,29 @@ import FormPage from './pages/containers/FormPageContainer';
 import SelectedBerthPage from './pages/containers/SelectedBerthPageContainer';
 import ThankYouPage from './pages/containers/ThankYouPageContainer';
 
+import SelectedAreasPage from './pages/containers/SelectedAreasPageContainer';
+import WinterBerthPage from './pages/containers/WinterBerthPageContainer';
+import WinterFormPage from './pages/containers/WinterFormPageContainer';
+
 export interface Props {
   locale: 'fi' | 'en' | 'sv';
 }
 
+const localeParam = ':locale(fi|en|sv)';
```

Comment:
But why?

Reply:
So it doesn't accept other values and goes to "not-found" page